src/Files.App/Actions/Global/FindAction.cs

```diff
@@ -0,0 +1,70 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Commands;
+using Files.App.Contexts;
+using Files.App.Extensions;
+using Files.App.Views;
+using Files.App.Views.LayoutModes;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using Windows.System;
+
+namespace Files.App.Actions
+{
+	internal class FindAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		public string Label => "Search".GetLocalizedResource();
+
+		public string Description => "TODO: Need to be described.";
+
+		public HotKey HotKey { get; } = new(VirtualKey.F, VirtualKeyModifiers.Control);
+
+		public HotKey SecondHotKey { get; } = new(VirtualKey.F3);
+
+		public RichGlyph Glyph { get; } = new();
+
+		public bool IsExecutable => context.ShellPage is not null && IsPageTypeValid();
```

Comment:
Search is available on any page so I think we can remove the IsExecutable property.

Reply:
I meant that we had some bugs caused by `context.ShellPage` being null so it might be better to leave it, or at least I'll add a null check in `ExecuteAsync()`

Reply:
👍